### PR TITLE
Add warning for import cmd

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

Add warning for import cmd

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/310
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/maintain-dm-using-tiup.md [hunk 1]
```diff
@@ -247,7 +247,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 
 > **Note:**
 >
-> TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
+>- TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
+>- Stop the original cluster before importing. 
+>- The deployment directories generated by some components are different from that of the original cluster. You can execute the `display` command to view the details.
```
Comment (july2993) [inline en/maintain-dm-using-tiup.md:252]: s/generated by/generated for/ ?

Comment (ireneontheway) [reply en/maintain-dm-using-tiup.md:252]:
```suggestion
>- The deployment directories of some components are different from that of the original cluster. You can execute the `display` command to view the details. 
```
Is it ok to use "of"? As the source text is:
> 部分组件生成的部署目录会跟原集群不一样，具体可以使用 `display` 命令查看。

Comment (july2993) [reply en/maintain-dm-using-tiup.md:252]: LGTM

**Review by july2993 — APPROVED** [review]

File: en/maintain-dm-using-tiup.md [hunk 2]
```diff
@@ -247,7 +247,10 @@ tiup dm patch prod-cluster /tmp/dm--hotfix.tar.gz -N 172.16.4.5:8261
 
 > **Note:**
 >
-> TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
+>- TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
+>- Stop the original cluster before importing. 
+>- The deployment directories of some components are different from that of the original cluster. You can execute the `display` command to view the details. 
+>- Run `tiup update --self && tiup update dm` before importing to make sure that the TiUP DM component is the latest version.
```
Comment (yikeke) [inline en/maintain-dm-using-tiup.md:253]:
```suggestion
> - TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
> - You need to stop the original cluster before importing. 
> - The deployment directories created by some components are different from those of the original cluster. You can execute the `display` command to view the details. 
> - Run `tiup update --self && tiup update dm` before importing to make sure that the TiUP DM component is the latest version.
```

**Review by yikeke — APPROVED** [review]
Rest LGTM

**Review by july2993 — COMMENTED** [review]
LGTM

**Review by july2993 — COMMENTED** [review]
LGTM

**Pull request merged**